### PR TITLE
Unit tests emma 30

### PR DESCRIPTION
**Pull request by Emma-MacK**
A basic start at testing for function used for making the bed files

### CODE REVIEW

File: docs/Running make bed tests.md
Comment (raymondmiles) [inline docs/Running make bed tests.md:1]: pip install pytest is necessary.
pytest should be added to requirements.txt

**Review by raymondmiles — COMMENTED** [review]
The tests are working well. 
requirements.txt should be updated to allow for the user to run the tests.

File: docs/Running make bed tests.md
Comment (Emma-MacK) [reply docs/Running make bed tests.md:1]: Done

**Review by raymondmiles — APPROVED** [review]
This works well and the requirements.txt has been updated.

**Pull request merged**